### PR TITLE
Docker setup

### LINKED ISSUE #21: Docker install and usage

### PR DESCRIPTION
**Pull request by stefanorosanelli**
This PR closes #21

### CODE REVIEW

File: source/setup.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Setup
 *****
 
+Below you will find instructions to setup BEdita4 on a tyipical server or desktop.
```
Comment (didoda) [inline source/setup.rst]: typo: `typical`

File: source/setup.rst [hunk 2]
```diff
@@ -163,3 +168,87 @@ Other noteworthy fields:
...
+.. code-block:: bash
+
+    $ docker run -p 8090:80 --env BEDITA_API_KEY=1029384756 \
+        --env BEDITA_ADMIN_USR=admin --env BEDITA_ADMIN_PWD=admin \
+        bedita/bedita:latest
+
+
+This will launch a BEdita4 instance using ``SQLite`` as its storage backend. It should become available at http://localhost:8090/home almost instantly.
+
+Replace ``bedita/bedita:latest`` with ``bedita4-local`` (or other chosen name) to lanch a local built image.
```
Comment (didoda) [inline source/setup.rst]: typo: `launch`

**Review by didoda — CHANGES_REQUESTED** [review]
minor typos

File: source/setup.rst (hunk 2 above)
Comment (stefanorosanelli) [reply source/setup.rst]: done

File: source/setup.rst (hunk 1 above)
Comment (stefanorosanelli) [reply source/setup.rst]: done

**Review by didoda — APPROVED** [review]

**Pull request merged**